controllers/csi/driver/server.go

```diff
@@ -60,7 +61,7 @@ var (
 		Namespace: "dynatrace",
 		Subsystem: "csi_driver",
 		Name:      "agent_versions",
-		Help:      "Number of an agent version currently mounted",
+		Help:      "Number of an agent version currently mounted by the CI driver",
```

Comment:
I guess you meant to write CSI driver, didn't you?

Reply:
yep 😅